### PR TITLE
[nginx-ingress, kube-prometheus] Templates for alerts

### PR DESCRIPTION
**Pull request by goruha**
## What
* Template prometheus alerts

## Why
* To have nice alerts

### CODE REVIEW

File: helmfile.d/0320.nginx-ingress.yaml [hunk 1]
```diff
@@ -149,11 +149,11 @@ releases:
           rules:
           - alert: NginxIngressControllerError
             annotations:
-              description: Nginx ingress controller error.
+              description: Nginx ingress controller error
               summary: NginxIngressControllerError
-            expr: nginx_ingress_controller_errors > 0
+            expr: nginx_ingress_controller_requests > 100000
```
Comment (aknysh) [inline helmfile.d/0320.nginx-ingress.yaml]: should we parameterize it?

File: helmfile.d/0320.nginx-ingress.yaml [hunk 2]
```diff
@@ -149,11 +149,11 @@ releases:
...
             labels:
-              severity: none
+              severity: warning
```
Comment (aknysh) [inline helmfile.d/0320.nginx-ingress.yaml]: should we parameterize it?

File: helmfile.d/0400.kube-prometheus.yaml [hunk 3]
```diff
@@ -110,6 +110,8 @@ releases:
                   ### Required: KUBE_PROMETHEUS_ALERT_MANAGER_SLACK_CHANNEL; e.g. #alerts-staging
                   channel: '{{ env "KUBE_PROMETHEUS_ALERT_MANAGER_SLACK_CHANNEL" }}'
                   send_resolved: true
+          templates:
+            - ./*.tmpl
```
Comment (aknysh) [inline helmfile.d/0400.kube-prometheus.yaml:114]: where do we have those `.tmpl` files?

**Review by aknysh — COMMENTED** [review]
see comments

File: helmfile.d/0320.nginx-ingress.yaml (hunk 2 above)
Comment (goruha) [reply helmfile.d/0320.nginx-ingress.yaml]: nope

File: helmfile.d/0400.kube-prometheus.yaml (hunk 3 above)
Comment (goruha) [reply helmfile.d/0400.kube-prometheus.yaml:114]: Here
https://github.com/cloudposse/helmfiles/pull/48/files#diff-e6ab6531a3930583484f9a763ca1fd7b
https://github.com/cloudposse/helmfiles/pull/48/files#diff-4d017cfa59e63b8a332ebe7374104e48R163

**Review by aknysh — APPROVED** [review]

**Review by osterman — APPROVED** [review]

**Pull request merged**